components/FormikFields/PaperInputPicker/index.js

```diff
@@ -502,6 +569,29 @@ const PaperInputPicker = ({
           )}
         </View>
       )}
+      {fieldType === 'loop' && (
+        <View key={formikKey}>
+          {additionalQuestions !== undefined && additionalQuestions.length !== 0
+          && additionalQuestions.map((question) => (
+            <PaperInputPicker
+              data={question}
+              formikProps={formikProps}
+              customForm={customForm}
+              config={config}
+              loopsAdded={loopsAdded}
+              setLoopsAdded={setLoopsAdded}
+            />
+          ))}
+          <Button onPress={() => addLoop()}>
+            {I18n.t('paperButton.addAdditional')}{translatedLabel}
+          </Button>
+          {individualLoopsAdded !== 0 && (
+            <Button onPress={() => removeLoop()}>
+              {I18n.t('paperButton.removePrevious')}{translatedLabel}
+            </Button>
+          )}
+        </View>
+      )}
     </>
```

Comment:
`<Looper />` component would replace all this logic

and all the state management logic would also be moved to the `looper.js`

Reply:
I'll replace the above portion and this into a looper component